definitions/npm/formik_v0.11.x/flow_v0.53.x-/formik_v0.11.x.js

```diff
@@ -184,7 +199,7 @@ declare module "formik" {
       /** Value of the input */
       value: any,
       /* name of the input */
-      name?: string,
+      name?: string
```

Comment:
Hello, I found another issue that we could maybe bundle into this PR?

https://github.com/jaredpalmer/formik/blob/v0.11.11/src/Field.tsx#L42 - According to the typescript definiation name is not optional.

Ciaran

Reply:
Sure, added the change in.